apiserver/facades/client/client/bundles.go

```diff
@@ -15,12 +15,14 @@ import (
 // applied in order.
 // This call is deprecated, clients should use the GetChanges endpoint on the
 // Bundle facade.
+// Note: any new feature in the future like devices will never be supported here.
 func (c *Client) GetBundleChanges(args params.BundleChangesParams) (params.BundleChangesResults, error) {
 	st := c.api.state()
 
 	bundleAPI, err := bundle.NewBundleAPI(st, c.api.auth, names.NewModelTag(st.ModelUUID()))
 	if err != nil {
 		return params.BundleChangesResults{}, err
 	}
-	return bundleAPI.GetChanges(args)
+	apiV1 := bundle.APIv1{&bundle.APIv2{bundleAPI}}
```

Comment:
That's not right. This call should not have changed. Instead your bundle.NewBundleAPI should return v2.

Reply:
The `NewBundleAPI` returns `BundleAPI`, but facade constructors return `APIv*`

Reply:
Right, I see. So this functionality exists on both Client and Bundle facade. And the one on the Client facade should behave as previously whereas the on Bundle facade has been revved up...

So this call should probably be bundle.NewFacadeV1().

Reply:
By the way, if here is the only palce that you need the data returned to be different (say with no devices), then why can't we just change the output here once we get the return from bundleAPI.GetChanges()? Would it not also avoid you making a PR against bundelchanges repo?